cmake/modules/RootBuildOptions.cmake

```diff
@@ -221,7 +221,6 @@ if(all)
  set(qt5web_defvalue ON)
  set(r_defvalue ON)
  set(roofit_defvalue ON)
- set(webui_defvalue ON)
```

Comment:
I would leave it with the fixed name: `set(webgui_defvalue ON)`. If we decide not to initialize it to `root7` later for any reason, `-Dall=ON` will still try to enable it.